### DISCUSSION TITLE
Check if users can automatically assigned somehow Define a mapping table of ADO users to GitHub?

### DISCUSSION THREAD

**Discussion by Sinnovate** [ahacke/ado-workitems-to-github-issues#41] [Q&A]
Check if users can automatically assigned somehow
Define a mapping table of ADO users to GitHub?

Kindly share the code to mape user permissions while migrating from ADO to GitHub

**Comment by ahacke** [comment]
Hi @Sinnovate,

as you can see in the readme of the repository, user mapping is part of the todo list. So I have not looked into that in detail yet. 

So far, I only have an idea for assigning users in GitHub based on information from Azure DevOps. The approach might vary if you're using GitHub Enterprise Managed Users (GHEMU). Generally, the concept involves creating a mapping table of user data from Azure DevOps to GitHub and using this as input for the migration script, which will enable user assignment at the end.

This covers not exactly your request regarding permissions though. What exactly do you have in mind? Do you want to create the same permission setup you have in ADO for GitHub? If yes, if you can fetch the data from via the API of ADO, that should be possible. But as I mentioned before, I have not worked on the user migration yet.

**Reply by Sinnovate** [reply]
Yes, I want a similar permission setup for the users as well as the mapping of users to GitHub. Thank you for your reply.
